customersupport/customersupport_tests.py

```diff
@@ -0,0 +1,153 @@
+            assert actual.json() == expected, "actual: " + str(actual.json())
+
+#tests for Sales Customer
+    def test_sales_customer(self):
+        with requests_mock.mock() as m:
+            StubberSalesCustomer.stubMock(m)
+            c_first = ""
+            try:
+                actual = sales.search_customer("Joe")
+                c_first = actual[0].first_name
+            except ConnectionError:
+                actual = "Joe"
+            expected = "Joe"
+            assert c_first == expected
+
+#Tests to check if search_customer was called
+    def test_sales_customer_called(self):
+        with requests_mock.mock() as m:
+            StubberSalesCustomer.stubMock(m)
+            sales.search_customer("Joe")
+            actual = m.called
+            expected = True
+            assert actual == expected, "actual: " + str(actual.json())
+
+# Cant put data into sales DB
+    """def test_sales_refund(self):
+        with requests_mock.mock() as m:
+            StubberSalesReturn.stubMock(m)
+            actual = sales.initiate_refund(True,1,1)
+            expected = {}
+            assert actual.json() == expected, "actual:" + str(actual.json())"""
+
+# API for returns doesn't exist yet so should be 404
+    def test_sales_refund_called(self):
+        '''with requests_mock.mock() as m:
+            StubberSalesReturn.stubMock(m)
+            sales.initiate_refund(True,1,1)
+            actual = m.called
+            expected = True
+            assert actual == expected, "actual: " + str(actual.json())'''
+
+#tests for Sales order lookup Empty dic since there are currently 0 orders
+    def test_sales_orders(self):
+        with requests_mock.mock() as m:
+            StubberSalesOrder.stubMock(m)
+            actual = sales.get_orders("John doe 1111 street Rochester NY 14568")
+            expected = []
+            assert actual == expected
+
+    def test_sales_orders_called(self):
+        with requests_mock.mock() as m:
+            StubberSalesOrder.stubMock(m)
+            sales.get_orders("John doe 1111 street Rochester NY 14568")
+            actual = m.called
+            expected = True
+            assert actual == expected, "actual: " + str(actual.json())
+
+#tests for Sales order information
+    def test_sales_orderinfo(self):
+        with requests_mock.mock() as m:
+            final = []
+            StubberSalesOrderInfo.stubMock(m)
+            actual = sales.get_order_info(1)
+            final = [actual[0].id, actual[0].order_date, actual[0].items]
+            expected = [1, '2017-03-01T20:51:26.905Z', set()]
+            assert final == expected
+
+    '''def test_sales_orderinfo_called(self):
+        with requests_mock.mock() as m:
+            StubberSalesOrderInfo.stubMock(m)
+            sales.get_order_info(1)
+            actual = m.call
+            expected = True
+            assert actual == expected, "actual: " + str(actual.json())'''
+
+#check to see if server is alive
```

Comment:
For comments like this, describing the tests, please put them as PEP8-compliant method docstrings.
[Example](https://www.python.org/dev/peps/pep-0257/#id16)